package.json

```diff
@@ -21,24 +21,26 @@
   "peerDependencies": {
     "d3-selection": "^1.4.1",
     "d3-transition": "^1.3.2",
-    "react": "^16.12.0"
+    "react": "^16.12.0 || 17.x || 18.x"
```

Comment:
@ivanross I'd propose just being optimistic and just putting >16, what would you say??